Final Project/team.py

```diff
@@ -7,7 +7,7 @@
 
 
 def teams(Id, value=5):
-    def predict_score(player, position, date):  # position: batsman, all-rounder, wk, bowler
+    def predict_score(player, position, date):
         # date format yyyy-mm-dd
         # When assigning position, make sure the file exists in github repo
```

Comment:
Delete this comment line

Reply:
the comment line," #when assigning position" has to be deleted

Reply:
Alright!